packages/vuetify/src/composables/transition.ts

```diff
@@ -0,0 +1,13 @@
+// Utilities
+import propsFactory from '@/util/propsFactory'
+
+// Types
+import type { PropType } from 'vue'
+
+export const makeTransitionProps = propsFactory({
+  transition: {
+    type: [Boolean, String] as PropType<string | false>,
```

Comment:
this does not have the object variant that I added. but it can be re-implemented in a separate PR